### PR TITLE
auto cap sentences. keep send / icons on the bottom always on multiline

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers quick styling fix
cc: @cecileboucheron

### CODE REVIEW

File: shared/chat/conversation/input/index.native.js [hunk 1]
```diff
@@ -115,13 +134,13 @@ const styleContainer = {
   alignItems: 'center',
   borderTopColor: globalColors.black_05,
   borderTopWidth: 1,
-  minHeight: 48,
+  paddingBottom: 10,
+  paddingTop: 10,
```
Comment (cecileboucheron) [inline shared/chat/conversation/input/index.native.js]: Can we use globalMargins.tiny here? (8px)

**Comment by keybase-ci-visdiff** [comment]
The commits 2e011d2a6f44...e1e2f42f4392 introduce visual changes on linux. <details><summary>:mag_right: 4 changed</summary>

 * Changed: **unlock_folders-Success** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/unlock_folders-Success-2e011d2a6f44.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/unlock_folders-Success-e1e2f42f4392.png)  
   unlock_folders-Success comparison
 * Changed: **unlock_folders-Normal** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/unlock_folders-Normal-2e011d2a6f44.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/unlock_folders-Normal-e1e2f42f4392.png)  
   unlock_folders-Normal comparison
 * Changed: **Tracker-Tracker_Error** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/Tracker-Tracker_Error-2e011d2a6f44.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/Tracker-Tracker_Error-e1e2f42f4392.png)  
   Tracker-Tracker_Error comparison
 * Changed: **Tracker-Platforms** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/Tracker-Platforms-2e011d2a6f44.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1494009466108-2e011d2a6f44-e1e2f42f4392-linux/Tracker-Platforms-e1e2f42f4392.png)  
   Tracker-Platforms comparison
</summary>

File: shared/chat/conversation/input/index.native.js [hunk 2]
```diff
@@ -94,16 +95,34 @@ class ConversationInput extends Component<void, Props, void> {
 
 const Action = ({text, onSubmit, editingMessage, openFilePicker, isLoading}) => (
   text ? (
-    <ClickableBox feedback={false} onClick={onSubmit}>
-      <Box style={{padding: globalMargins.small}}>
+    <ClickableBox feedback={false} onClick={onSubmit} style={{}}>
```
Comment (cjb) [inline shared/chat/conversation/input/index.native.js]: huh, what does the extra style={{}} do?

Comment (chrisnojima) [inline shared/chat/conversation/input/index.native.js]: nothing, leftover from me messing around, will remove

**Pull request merged**